mmv1/third_party/terraform/services/firebase/data_source_google_firebase_web_app_config_test.go.tmpl

```diff
@@ -0,0 +1,50 @@
+package firebase_test
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccDataSourceGoogleFirebaseWebAppConfig(t *testing.T) {
+	t.Parallel()
+
```

Comment:
This test needs to be skipped in VCR as the muxing fixes aren't merged into main, which means VCR tests combining SDK and PF implemented resources/data sources will fail.

```suggestion
func TestAccDataSourceGoogleFirebaseWebAppConfig(t *testing.T) {
	// TODO: https://github.com/hashicorp/terraform-provider-google/issues/14158
	acctest.SkipIfVcr(t)
	t.Parallel()
```